genienlp/data_utils/numericalizer.py

```diff
@@ -145,26 +167,24 @@ def save(self, save_dir):
         if len(self._special_tokens_to_word_map) > 0:
             with open(os.path.join(save_dir, 'special-token-preprocessing.json'), 'w') as fp:
                 json.dump(self._special_tokens_to_word_map, fp)
-
+    
     def build_vocab(self, vocab_sets, tasks):
-        self._tokenizer = AutoTokenizer.from_pretrained(self._pretrained_name,
-                                                        do_lower_case=False,
-                                                        do_basic_tokenize=False,
-                                                        cache_dir=self._cache,
-                                                        use_fast=self._use_fast())
-
         special_tokens = []
         for task in tasks:
             special_tokens += list(task.special_tokens)
         special_tokens.sort()
-
+        
         if self._preprocess_special_tokens:
             self._build_special_tokens_maps(special_tokens)
             self._build_special_tokens_regexes()
         else:
             # add the special tokens directly to the tokenizer
             self._tokenizer.add_tokens(special_tokens)
-
+        
+        # add entity boundary special tokens
+        if self.args.add_types_to_text != 'no':
+            self._tokenizer.add_tokens(['<e>', '</e>'])
```

Comment:
Have you experimented using regular tokens instead of special tokens as entity boundaries, given pretraining?

Reply:
I suspect it would make a difference cause adding types to text does not generate more `correct_without_param` errors but I can try.